frontend/src/hooks/auth/useGetIdentity.ts

```diff
@@ -28,26 +28,26 @@ const defaultUser: User = {
 };
 
 /**
- * Return the current user identity by calling authProvider.getIdentity() on mount
+ * Return the current user identity
  *
  * The return value updates according to the call state:
  *
  * - mount: { loading: true, loaded: false }
  * - success: { identity: Identity, loading: false, loaded: true }
  * - error: { error: Error, loading: false, loaded: true }
  *
- * The implementation is left to the authProvider.
- *
  * @returns The current user identity. Destructure as { identity, error, loading, loaded }.
  */
 const useGetIdentity = () => {
-  const { data, ...rest } = useSafeRequest(getUser, {
+  const { data, loading, error } = useSafeRequest(getUser, {
     cacheKey: CacheKey.User,
   });
 
   return {
     identity: data?.data ?? defaultUser,
-    ...rest,
+    loading,
```

Comment:
Have u tested this doesn't cause things to break on other places?

Reply:
Yes. tested 👍